### ISSUE TITLE
Iteration Plan for <Month> 2022 [DRAFT]

### ISSUE THREAD

**Issue by kieferrm** [microsoft/vscode#151843]
This plan captures our work in **June**. This is a 4-week iteration. We will ship in early **July**.

## Endgame
- **June 27, 2022**: Endgame begins
- **July 1, 2022**: Endgame done

The endgame details for this iteration are tracked [here](https://github.com/microsoft/vscode/issues?q=is%3Aissue+label%3Aendgame-plan+milestone%3A%22June+2022%22).

## Plan Items

Below is a summary of the top level plan items.

Legend of annotations:

| Mark                 | Description                                       |
| -------------------- | ------------------------------------------------- |
| :runner:             | work in progress                                  |
| :hand:               | blocked task                                      |
| :muscle:             | stretch goal for this iteration                   |
| :red_circle:         | missing issue reference                           |
| :large_blue_circle:  | more investigation required to remove uncertainty |
| :black_circle:       | under discussion within the team                  |
| :black_large_square: | a large work item, larger than one iteration      |

### Accessibility
- [ ] :runner: Accessibility issues, see [query](https://github.com/Microsoft/vscode/issues?q=is%3Aopen+is%3Aissue+label%3Aaccessibility+milestone%3A%22June+2022%22) @isidorn **team**

### UX
- [ ] :runner: Improve the discoverability of filtering in tree views [vscode#146806](https://github.com/microsoft/vscode/issues/146806) @misolori @joaomoreno
- [ ] :muscle: Explore "Do Not Disturb" mode and its implications on existing scenarios [vscode#144324](https://github.com/microsoft/vscode/issues/144324) @daviddossett @bpasero
- [ ] :red_circle: :muscle: Improve editor group border contrast @daviddossett

### Workbench
- [ ] :runner: Support profile management [vscode#116740](https://github.com/microsoft/vscode/issues/116740) @sana-ajani @sandy081 @daviddossett
- [ ] Profiles: Support global state isolation per profile [vscode#151490](https://github.com/microsoft/vscode/issues/151490) @bpasero @sandy081
- [ ] :red_circle: :muscle: Support hiding toolbar items using right click @jrieken
- [ ] :runner: Explore `Continue On...` support in core [vscode#141293](https://github.com/microsoft/vscode/issues/141293#issuecomment-1117468466) @joyceerhl
- [ ] :runner: Explore a `Share as...` menu for creating `vscode.dev` links [vscode#146309](https://github.com/microsoft/vscode/issues/146309) @alexr00
- [ ] :runner: :muscle: Adopt inline suggestions for previewing emmet expansions [vscode#139247](https://github.com/microsoft/vscode/issues/139247) @rzhao271 @hediet
- [ ] :runner: Polish Command Palette launcher in titlebar [vscode#149445](https://github.com/microsoft/vscode/issues/149445) @jrieken @sbatten
- [ ] :runner: Settings Editor: Improve setting override indicators UX when many items are present [vscode#151787](https://github.com/microsoft/vscode/issues/151787) @rzhao271
- [ ] :runner: Settings Editor: Indicate if setting has a language-specific value [vscode#145731](https://github.com/microsoft/vscode/issues/145731)  @rzhao271
- [ ] :hand: Adopt Window Control Overlay API in electron 18 [vscode#147947](https://github.com/microsoft/vscode/pull/147947) @sbatten

### Web
- [ ] :runner: Continue translations in the web [vscode#82595](https://github.com/microsoft/vscode/issues/82595) @tylerleonhardt
  - [ ] Support built-in extension translations in web
  - [ ] Support marketplace extenson translations in web
- [ ] :runner: Explore how a web extension can leverage the [WebUSB API](https://wicg.github.io/webusb) [vscode#116761](https://github.com/microsoft/vscode/issues/116761) @mjbvz @alexdima
- [ ] :red_circle: Support default themes in `/theme` @aeschli
- [ ] :runner: Explore a partial WASI implementation using the VS Code API [vscode#149092](https://github.com/microsoft/vscode/issues/149092) @dbaeumer
- [ ] :red_circle: Explore setting COOP and COEP headers on vscode.dev @jrieken

### Code Editor
- [ ] :runner: Merge editor [vscode#146091](https://github.com/microsoft/vscode/issues/146091) @hediet @jrieken @bpasero
- [ ] Context menu on the minimap to hide/show it [vscode#148131](https://github.com/microsoft/vscode/issues/148131) @alexdima
- [ ] Investigate providing a default snippet for a language [vscode#145929](https://github.com/microsoft/vscode/issues/145929) @jrieken @digitarald
- [ ] :red_circle: Eliminate notification that shows when there are too many folding regions @aeschli

### Comments
- [ ] Allow to disable comments altogether [vscode#149176](https://github.com/microsoft/vscode/issues/149176) @alexr00
- [ ] :muscle: Comments view should have ability to filter by resolved/unresolved [vscode#150958](https://github.com/microsoft/vscode/issues/150958) @alexr00

### Notebook Editor
- [ ] Retain controller on `saveAs` or `save` of an untitled notebook [vscode#151158](https://github.com/microsoft/vscode/issues/151158) @roblourens
- [ ] :runner: Improve notebook “scrolling up” rendering performance [vscode#148164](https://github.com/microsoft/vscode/issues/148164) @roblourens
- [ ] :muscle: Explore improving files search results for notebooks [vscode#95275](https://github.com/microsoft/vscode/issues/95275) @roblourens
- [ ] Investigate support for hot exit in interactive window [vscode#151444](https://github.com/microsoft/vscode/issues/151444) @lramos15 @rebornix
- [ ] :red_circle: Improve selecting a renderer when multiple are registered for the same output mime-type @rebornix

#### Jupyter Notebooks
- [ ] :red_circle: Explore improve first-run experience for running a Jupyter notebook @brettcannon @lramos15 @IanMatthewHuff @karrtikr @minsa110
- [ ] :red_circle: :runner: Contributions to the vscode-jupyter extension @greazer **team**

#### AnyCode
- None.

#### LSP
- None.

#### Python
- [ ] :red_circle: Contributions to the python extension @brettcannon **team**
- [ ] :red_circle: Resolve language server restart issues with `pylance` @dbaeumer

#### JavaScript/TypeScript
- [ ] :hand: Support TS server plugins on web [vscode#140455](https://github.com/microsoft/vscode/issues/140455) @mjbvz
- [ ] :hand: Support TS project-wide intellisense on web [typescript#47600](https://github.com/microsoft/TypeScript/issues/47600) @mjbvz
- [ ] :hand: Explore automatic type acqusition on web [typescript#45314](https://github.com/microsoft/TypeScript/issues/45314) @mjbvz

#### Markdown
- [ ] :runner: Finalize DropIntoEditor API for creating file and image links [vscode#142990](https://github.com/microsoft/vscode/issues/142990) @mjbvz
- [ ] :runner: Finalize DocumentPasteEditProvider API to support pasting images [vscode#30066](https://github.com/microsoft/vscode/issues/30066) @mjbvz

#### CSS
- None.

#### JSON
- [ ] :muscle: Explore test suite from JSON schema spec [vscode-json-languageservice#120](https://github.com/microsoft/vscode-json-languageservice/issues/102) @aeschli

### Debug
- [x] :runner: Show breakpoint diagnostic information in the breakpoints view [vscode#142870](https://github.com/microsoft/vscode/issues/142870) @roblourens
- [ ] Support to pass unescaped arguments in launch.json [vscode#149910](https://github.com/microsoft/vscode/issues/149910) @roblourens @weinand
- [ ] Improving the UI for "step in targets" [vscode#149871](https://github.com/microsoft/vscode/issues/149871) @roblourens 
- [ ] :runner: Remove special treatment of extension debugging in VS Code (Prototype extension debugging in a clean environment) [vscode#126737](https://github.com/microsoft/vscode/issues/126737) @weinand

#### DAP
- [ ] Support to pass unescaped program arguments [debug-adapter-protocol#146](https://github.com/microsoft/debug-adapter-protocol/issues/146) @weinand @roblourens
- [ ] Add additional location information to StepInTarget [debug-adapter-protocol#274](https://github.com/microsoft/debug-adapter-protocol/issues/274) @connor4312 @weinand

#### js-debug
- [ ] :runner: :muscle: Investigate: Smart Step Into [vscode#123879](https://github.com/microsoft/vscode/issues/123879) @connor4312
- [ ] Allow easy toggling between sourcemaps and compiled code [vscode#151412](https://github.com/microsoft/vscode/issues/151412) @connor4312
- [ ] :muscle: Improve breakpoint predictor caching for large projects [vscode-js-debug#1149](https://github.com/microsoft/vscode-js-debug/issues/1149) @connor4312
- [ ] :muscle: Automatically create debug config with a "dry run" [vscode-js-debug#1214](https://github.com/microsoft/vscode-js-debug/issues/1214) @connor4312

### Testing
- None.

### Terminal
- [ ] :red_circle: Prepare to make shell integration the default @tylerleonhardt @meganrogge
- [ ] :red_circle: Address issues with terminal reconnection @tyriar
- [ ] Improve accessibility in the terminal [vscode#149882](https://github.com/microsoft/vscode/issues/149882#issuecomment-1134946927) @tyriar
- [ ] :red_circle: Explore contextual actions in the terminal @tyriar

### Tasks
- [ ] :runner: Change feature area ownership of tasks [vscode#149060](https://github.com/microsoft/vscode/issues/149060) @meganrogge @tyriar @alexr00
- [ ] Explore reconnecting tasks on window reload [vscode#117408](https://github.com/microsoft/vscode/issues/117408) @meganrogge
- [ ] Support configurable icons and color for tasks [vscode#145965](https://github.com/microsoft/vscode/issues/145965) @meganrogge

### Source Control
- [x] :runner: Full editor can be used as git commit message editor [vscode#30562](https://github.com/microsoft/vscode/pull/30562) @lszomoru
- [ ] :muscle: Investigate: Support input box spell checking [vscode#35571](https://github.com/microsoft/vscode/issues/35571) @lszomoru

### API
- API finalization: [query](https://github.com/Microsoft/vscode/issues?q=is%3Aissue+is%3Aopen+milestone%3A%22June+2022%22+label%3A%22api-finalization%22) @jrieken @mjbvz
- API proposals: [query](https://github.com/Microsoft/vscode/issues?q=is%3Aissue+is%3Aopen+milestone%3A%22June+2022%22+label%3A%22api-proposal%22) @jrieken @mjbvz
- [ ] :red_circle: :muscle: Support snippets in workspace edits @jrieken
- [ ] :runner: Continue finalization of remaining proposed notebook API [vscode#149861)](https://github.com/microsoft/vscode/issues/149861) @mjbvz @rebornix @roblourens
- [ ] Finalize API to disable the Source Control input field [vscode#150268](https://github.com/microsoft/vscode/issues/150268) @lszomoru

### Extensions
- [x] :runner: Provide a filter to list all deprecated extensions [vscode#151729](https://github.com/microsoft/vscode/issues/151729) @sandy081

### Extension Contributions
- [ ] GitHub Repositories: [query](https://github.com/microsoft/vscode-remote-repositories-github/milestone/15) @joyceerhl
- [ ] :red_circle: Improvements to the Remote extensions @tanhakabir @chrmarti @aeschli
- [ ] :runner: GHPR: Improvements papercuts, selfhosting issues, [query](https://github.com/microsoft/vscode-pull-request-github/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22June+2022%22) @alexr00
- [ ] Close the loop between committing and creating a PR [vscode#151589](https://github.com/microsoft/vscode/issues/151589) @alexr00 @lszomoru

### Install / Update
- [ ] Windows: Unable to update [vscode#147408](https://github.com/microsoft/vscode/issues/147408) @joaomoreno
- [ ] Windows: Prevent update service from activating if running as Admin in User install [vscode#148428](https://github.com/microsoft/vscode/issues/148428) @joaomoreno
- [ ] :runner: Integrate with the Windows 11 explorer context menu [vscode#127365](https://github.com/microsoft/vscode/issues/127365) @deepak1556

### Engineering
- [ ] Sandbox: enable extension host communication [vscode#135268](https://github.com/microsoft/vscode/issues/135268) @deepak1556 @alexdima
- [ ] :runner: Review errors to identify which ones are actionable as error telemetry [vscode#145043](https://github.com/microsoft/vscode/issues/145043) @lramos15 **team**
- [ ] :red_circle: Explore spliting compilation and transpilation into two distinct gulp tasks @jrieken
- [ ] Explore using `swc` for transpilation [vscode#150025](https://github.com/microsoft/vscode/issues/150025) @jrieken
- [x] :runner: Execute tests in parallel in CI build [vscode#151740](https://github.com/microsoft/vscode/issues/151740) @lszomoru
- [ ] :muscle: Enable `contextIsolation` for all windows [vscode#128099](https://github.com/microsoft/vscode/issues/128099) @bpasero @rzhao271 @deepak1556

#### Electron
- [x] :runner: Electron 18 update [vscode#145527](https://github.com/microsoft/vscode/pull/145527) @deepak1556
- [ ] :runner: Improve dependency generation for Debian [vscode#13089](https://github.com/microsoft/vscode/issues/13089) @rzhao271
- [ ] WCO on Windows don't respond to touch events [electron#34364](https://github.com/electron/electron/issues/34364) @deepak1556
- [ ] WCO on Windows occludes dev tools [electron#34129](https://github.com/electron/electron/issues/34129) @deepak1556

### Documentation
- [ ] :runner: Improve TS docs [vscode-docs#5205](https://github.com/microsoft/vscode-docs/issues/5205) @gregvanl @isidorn

----
### Deferred
- [ ] Allow resizing for the find widget in notebooks [vscode#141516](https://github.com/microsoft/vscode/issues/141516) @rebornix
- [ ] Support copying simple output to the clipboard [vscode#129880](https://github.com/microsoft/vscode/issues/129880) @amunger @rebornix
- [ ] Show changes in the overview ruler of the notebook diff editor [vscode#146224](https://github.com/microsoft/vscode/issues/146224) @amunger @rebornix
- [ ] :runner: Provide support for Go to Test/Go to Test Subject [vscode#126932](https://github.com/microsoft/vscode/issues/126932) @connor4312
- [ ] Indicate focused tab when using multiple editor groups in HC themes [vscode#145563](https://github.com/microsoft/vscode/issues/145563) @daviddossett
- [ ] Align editor find widget with the notebook find widget [vscode#144851](https://github.com/microsoft/vscode/issues/144851) @rebornix
- [ ] Make the eslint extension more approachable for new contributors [vscode-eslint#1447](https://github.com/microsoft/vscode-eslint/issues/1447) @dbaeumer
- [ ] Investigate eslint notebook support in the web [vscode-eslint#1366](https://github.com/microsoft/vscode-eslint/issues/1366) @dbaeumer
- [ ] Draft Git Beginner's Docs [vscode-docs#5033](https://github.com/microsoft/vscode-docs/issues/5033) @digitarald @burkeholland

**Comment by organicghemp** [comment]
> Este plan captura nuestro trabajo en **junio**. Esta es una iteración de 4 semanas. Enviaremos a principios de **julio**.
> 
> ## Final
> * **27 de junio de 2022**: Comienza Endgame
> * **1 de julio de 2022**: Endgame hecho
> 
> Los detalles del final del juego para esta iteración se rastrean [aquí](https://github.com/microsoft/vscode/issues?q=is%3Aissue+label%3Aendgame-plan+milestone%3A%22June+2022%22).
> 
> ## Elementos del plan
> A continuación se muestra un resumen de los elementos del plan de nivel superior.
> 
> Leyenda de anotaciones:
> 
> Marcar	Descripción
> 🏃	trabajos en curso
> ✋	tarea bloqueada
> 💪	objetivo de estiramiento para esta iteración
> 🔴	Falta referencia de problema
> 🔵	Se requiere más investigación para eliminar la incertidumbre
> ⚫	en discusión dentro del equipo
> ⬛	un elemento de trabajo grande, mayor que una iteración
> ### Accesibilidad
> * [ ]  Problemas de accesibilidad, consulte consulta @isidorn **equipo**
> 
> ### UX
> * [ ]  🏃 Mejore la capacidad de detección del filtrado en vistas de árbol [vscode#146806](https://github.com/microsoft/vscode/issues/146806) @misolori @joaomoreno
> * [ ]  💪 Explore el modo "No molestar" y sus implicaciones en los escenarios existentes [vscode#144324](https://github.com/microsoft/vscode/issues/144324) @daviddossett @bpasero
> * [ ]  💪 Mejore los estados de enfoque del grupo del editor en temas de alto contraste [vscode#145563](https://github.com/microsoft/vscode/issues/145563) @daviddossett
> 
> ### Mesa de trabajo
> * [ ]  🏃 Administración de perfiles de soporte [vscode#116740](https://github.com/microsoft/vscode/issues/116740) @sandy081 @sana-ajani @daviddossett
> * [ ]  🏃 Perfiles: Admite el aislamiento de estado global por perfil [vscode#151490](https://github.com/microsoft/vscode/issues/151490) @bpasero @sandy081
> * [ ]  🔴 💪 Admite la ocultación de elementos de la barra de herramientas con el botón derecho del ratón @jrieken
> * [ ]  🏃 Explore el soporte en el @joyceerhl core [vscode#141293](https://github.com/microsoft/vscode/issues/141293#issuecomment-1117468466)`Continue On...`
> * [ ]  🏃 Explore un menú para crear enlaces [vscode#146309](https://github.com/microsoft/vscode/issues/146309) @alexr00`Share as...``vscode.dev`
> * [x]  🏃 💪 Adopte sugerencias en línea para obtener una vista previa de las expansiones emmet [vscode#139247](https://github.com/microsoft/vscode/issues/139247) @rzhao271 @hediet
> * [ ]  🏃 Lanzador de paleta de comandos polaco en titlebar [vscode#149445](https://github.com/microsoft/vscode/issues/149445) @jrieken @sbatten
> * [ ]  🏃 Editor de configuración: Mejore la configuración de los indicadores de anulación UX cuando hay muchos elementos presentes [vscode#151787](https://github.com/microsoft/vscode/issues/151787) @rzhao271
> * [ ]  🏃 Editor de configuración: Indique si la configuración tiene un valor específico del idioma [vscode#145731](https://github.com/microsoft/vscode/issues/145731) @rzhao271
> * [ ]  ✋ Adopte la API de superposición de control de ventana en electron 18 [vscode#147947](https://github.com/microsoft/vscode/pull/147947) @sbatten
> * [ ]  Habilitar la compatibilidad con directivas de grupo para el @joaomoreno [vscode#147756](https://github.com/microsoft/vscode/issues/147756)`UpdateMode`
> * [ ]  🏃 Utilice el editor de combinación para resolver conflictos en la configuración de sincronización [vscode#151940](https://github.com/microsoft/vscode/issues/151940) @sandy081
> 
> ### Telaraña
> * [ ]  🏃 Continuar traducciones en la web [vscode#82595](https://github.com/microsoft/vscode/issues/82595) @TylerLeonhardt
>   
>   * [ ]  Admite traducciones de extensiones integradas en la web
>   * [ ]  Soporte de traducciones de extenson de mercado en web
> * [ ]  🏃 Explore cómo una extensión web puede aprovechar la [API de WebUSB](https://wicg.github.io/webusb) [vscode#116761](https://github.com/microsoft/vscode/issues/116761) @mjbvz @alexdima
> * [ ]  🔴 Admite temas predeterminados en @aeschli`/theme`
> * [ ]  🏃 Explore una implementación parcial de WASI utilizando la API [vscode#149092](https://github.com/microsoft/vscode/issues/149092) de VS Code @dbaeumer
> * [ ]  Explore la configuración de encabezados COOP y COEP en vscode.dev [vscode#137884](https://github.com/microsoft/vscode/issues/137884) @jrieken
> 
> ### Editor de código
> * [ ]  🏃 Editor de [combinación vscode#146091](https://github.com/microsoft/vscode/issues/146091) @bpasero @jrieken @hediet
> * [ ]  Menú contextual en el minimapa para ocultarlo/mostrarlo [vscode#148131](https://github.com/microsoft/vscode/issues/148131) @alexdima
> * [ ]  Investigar la provisión de un fragmento de código predeterminado para un @jrieken [de @jrieken de idioma vscode#](https://github.com/microsoft/vscode/issues/145929)  145929 @digitarald
> * [ ]  🔴 Elimine la notificación que muestra cuando hay demasiadas regiones plegables @aeschli
> * [x]  Agregar comando para obtener una vista previa de las refactorizaciones [vscode#151140](https://github.com/microsoft/vscode/issues/151140) @justschen
> * [ ]  🔴 🏃 Comience a explorar el uso de un widget personalizado para acciones de código @justschen
> 
> ### Comentarios
> * [ ]  Permitir deshabilitar los comentarios por completo [vscode#149176](https://github.com/microsoft/vscode/issues/149176) @alexr00
> * [ ]  💪 La vista de comentarios debe tener la capacidad de filtrar por [vscode#150958](https://github.com/microsoft/vscode/issues/150958) resuelto/no resuelto @alexr00
> 
> ### Editor de cuadernos
> * [ ]  Conservar el controlador en o de un portátil sin título [vscode#151158](https://github.com/microsoft/vscode/issues/151158) @roblourens`saveAs``save`
> * [ ]  🏃 Mejore el rendimiento de renderizado de "desplazamiento hacia arriba" del bloc de notas [vscode#148164](https://github.com/microsoft/vscode/issues/148164) @roblourens
> * [ ]  💪 Explore la mejora de los resultados de búsqueda de archivos para blocs [de notas vscode#95275](https://github.com/microsoft/vscode/issues/95275) @roblourens
> * [ ]  Investigar la compatibilidad con la salida en caliente en la ventana interactiva [vscode#151444](https://github.com/microsoft/vscode/issues/151444) @lramos15 @rebornix
> * [ ]  🔴 Mejorar la selección de un renderizador cuando se registran varios para el mismo tipo mime de salida @rebornix
> 
> #### Cuadernos Jupyter
> * [ ]  🏃 Explore la mejora de la experiencia de primera ejecución para ejecutar un bloc de notas de Jupyter; [tablero](https://github.com/orgs/microsoft/projects/359/views/1); @brettcannon @minsa110 @lramos15 @karrtikr  @IanMatthewHuff @karrtikr
> * [ ]  🏃 Contribuciones al **equipo** de @greazer de la extensión [vscode-jupyter#10430](https://github.com/microsoft/vscode-jupyter/issues/10430)
> 
> ### Idiomas
> #### AnyCode
> * Ninguno.
> 
> #### LSP
> * Ninguno.
> 
> #### Pitón
> * [ ]  🏃 Contribuciones a la extensión [de python vscode-python#19286](https://github.com/microsoft/vscode-python/issues/19286) @brettcannon **equipo**
> * [ ]  🔴 Resolver problemas de reinicio del servidor de idiomas con @dbaeumer`pylance`
> 
> #### JavaScript/TypeScript
> * [ ]  ✋ Admite complementos de servidor TS en web [vscode#140455](https://github.com/microsoft/vscode/issues/140455) @mjbvz
> * [ ]  ✋ Admite intellisense en todo el proyecto de TS en [el tipo de letra web# 47600](https://github.com/microsoft/TypeScript/issues/47600) @mjbvz
> * [ ]  ✋ Explore la adquisición automática de tipos en la escritura web[#45314](https://github.com/microsoft/TypeScript/issues/45314) @mjbvz
> 
> #### Descuento
> * [ ]  🏃 Finalize DropIntoEditor API for creating file and image links [vscode#142990](https://github.com/microsoft/vscode/issues/142990) @mjbvz
> * [ ]  🏃 Finalize DocumentPasteEditProvider API to support pasting images [vscode#30066](https://github.com/microsoft/vscode/issues/30066) @mjbvz
> 
> #### CSS
> * None.
> 
> #### JSON
> * [ ]  💪 Explore test suite from JSON schema spec [vscode-json-languageservice#120](https://github.com/microsoft/vscode-json-languageservice/issues/102) @aeschli
> 
> ### Debug
> * [x]  🏃 Show breakpoint diagnostic information in the breakpoints view [vscode#142870](https://github.com/microsoft/vscode/issues/142870) @roblourens
> * [ ]  Support to pass unescaped arguments in launch.json [vscode#149910](https://github.com/microsoft/vscode/issues/149910) @roblourens @weinand
> * [ ]  Improving the UI for "step in targets" [vscode#149871](https://github.com/microsoft/vscode/issues/149871) @connor4312
> * [ ]  🏃 Remove special treatment of extension debugging in VS Code (Prototype extension debugging in a clean environment) [vscode#126737](https://github.com/microsoft/vscode/issues/126737) @weinand
> 
> #### DAP
> * [ ]  Support to pass unescaped program arguments [debug-adapter-protocol#146](https://github.com/microsoft/debug-adapter-protocol/issues/146) @weinand @roblourens
> * [ ]  Add additional location information to StepInTarget [debug-adapter-protocol#274](https://github.com/microsoft/debug-adapter-protocol/issues/274) @connor4312 @weinand
> 
> #### js-debug
> * [ ]  🏃 💪 Investigate: Smart Step Into [vscode#123879](https://github.com/microsoft/vscode/issues/123879) @connor4312
> * [x]  🏃 Allow easy toggling between sourcemaps and compiled code [vscode#151412](https://github.com/microsoft/vscode/issues/151412) @connor4312
> * [ ]  💪 Improve breakpoint predictor caching for large projects [vscode-js-debug#1149](https://github.com/microsoft/vscode-js-debug/issues/1149) @connor4312
> * [ ]  💪 Automatically create debug config with a "dry run" [vscode-js-debug#1214](https://github.com/microsoft/vscode-js-debug/issues/1214) @connor4312
> 
> ### Testing
> * None.
> 
> ### Terminal
> * [ ]  🏃 Prepare to make shell integration the default [vscode#151935](https://github.com/microsoft/vscode/issues/151935) @Tyriar @meganrogge
> * [ ]  🏃 Address issues with terminal reconnection [vscode#151936](https://github.com/microsoft/vscode/issues/151936) @Tyriar
> * [ ]  Improve accessibility in the terminal [vscode#151938](https://github.com/microsoft/vscode/issues/151938) @Tyriar
> * [ ]  🏃 Explore contextual actions in the terminal [vscode#151937](https://github.com/microsoft/vscode/issues/151937) @Tyriar
> 
> ### Tasks
> * [ ]  🏃 Change feature area ownership of tasks [vscode#149060](https://github.com/microsoft/vscode/issues/149060) @meganrogge @Tyriar @alexr00
> * [ ]  Explore reconnecting tasks on window reload [vscode#117408](https://github.com/microsoft/vscode/issues/117408) @meganrogge
> * [ ]  🏃 Support configurable icons and color for tasks [vscode#145965](https://github.com/microsoft/vscode/issues/145965) @meganrogge
> 
> ### Source Control
> * [x]  🏃 Full editor can be used as git commit message editor [vscode#30562](https://github.com/microsoft/vscode/pull/30562) @lszomoru
> * [ ]  💪 Investigate: Support input box spell checking [vscode#35571](https://github.com/microsoft/vscode/issues/35571) @lszomoru
> 
> ### API
> * API finalization: [query](https://github.com/Microsoft/vscode/issues?q=is%3Aissue+is%3Aopen+milestone%3A%22June+2022%22+label%3A%22api-finalization%22) @jrieken @mjbvz
> * API proposals: [query](https://github.com/Microsoft/vscode/issues?q=is%3Aissue+is%3Aopen+milestone%3A%22June+2022%22+label%3A%22api-proposal%22) @jrieken @mjbvz
> * [ ]  🔴 💪 Support snippets in workspace edits @jrieken
> * [ ]  🏃 Continuar la finalización de la @roblourens de @mjbvz de @mjbvz de API de bloc de notas [propuesta restante vscode#](https://github.com/microsoft/vscode/issues/149861)  @rebornix 149861) @roblourens
> * [ ]  Finalice la API para deshabilitar el campo de entrada de control [de código fuente vscode#150268](https://github.com/microsoft/vscode/issues/150268) @lszomoru
> 
> ### Extensiones
> * [x]  🏃 Proporcione un filtro para enumerar todas las extensiones obsoletas [vscode#151729](https://github.com/microsoft/vscode/issues/151729) @sandy081
> 
> ### Contribuciones de extensión
> * [ ]  Repositorios de GitHub: @joyceerhl [de consulta](https://github.com/microsoft/vscode-remote-repositories-github/milestone/15)
> * [ ]  Mejoras en las extensiones remotas [vscode-remote-release#6818](https://github.com/microsoft/vscode-remote-release/issues/6818) @tanhakabir @chrmarti @aeschli
> * [ ]  🏃 GHPR: Mejoras papercuts, problemas de autohospedaje, [consulta](https://github.com/microsoft/vscode-pull-request-github/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22June+2022%22) @alexr00
> * [ ]  Cierre el ciclo entre la confirmación y la creación de un [@lszomoru de pr vscode#](https://github.com/microsoft/vscode/issues/151589)151589 @alexr00
> 
> ### Instalar / Actualizar
> * [ ]  Windows: No se puede actualizar [el @joaomoreno vscode#](https://github.com/microsoft/vscode/issues/147408)147408
> * [ ]  Windows: Impedir que el servicio de actualización se active si se ejecuta como administrador en Instalación de usuario [vscode#148428](https://github.com/microsoft/vscode/issues/148428) @joaomoreno
> * [ ]  🏃 Integración con el menú contextual del explorador de Windows 11 [vscode#127365](https://github.com/microsoft/vscode/issues/127365) @deepak1556
> 
> ### Ingeniería
> * [ ]  Sandbox: habilitar la comunicación del host de extensión [vscode#135268](https://github.com/microsoft/vscode/issues/135268) @deepak1556 @alexdima
> * [ ]  🏃 Revise los errores para identificar cuáles son procesables como telemetría [de error vscode#145043](https://github.com/microsoft/vscode/issues/145043) @lramos15 **equipo**
> * [ ]  🔴 Explore la división de la compilación y la transpilación en dos tareas distintas de trago @jrieken
> * [ ]  Explore el uso de transpilación [vscode#150025](https://github.com/microsoft/vscode/issues/150025) @jrieken`swc`
> * [x]  🏃 Ejecute pruebas en paralelo en la compilación [de CI vscode#151740](https://github.com/microsoft/vscode/issues/151740) @lszomoru
> * [ ]  💪 Habilitar para todas las @deepak1556 [de @bpasero de @bpasero de windows vscode#](https://github.com/microsoft/vscode/issues/128099)  @rzhao271 128099`contextIsolation`
> 
> #### Electrón
> * [x]  🏃 Actualización de Electron 18 [vscode#145527](https://github.com/microsoft/vscode/pull/145527) @deepak1556
> * [ ]  🏃 Mejore la generación de dependencias para debian [vscode#13089](https://github.com/microsoft/vscode/issues/13089) @rzhao271
> * [ ]  La OMA en Windows no responde a los eventos [táctiles electron#34364](https://github.com/electron/electron/issues/34364) @deepak1556
> * [ ]  WCO en Windows ocluye las herramientas de desarrollo [electron#34129](https://github.com/electron/electron/issues/34129) @deepak1556
> 
> ### Documentación
> * [ ]  🏃 Mejorar los documentos de TS [vscode-docs#5205](https://github.com/microsoft/vscode-docs/issues/5205) @gregvanl @isidorn
> * [ ]  Revisar y actualizar Integrated Terminal [vscode-docs#4809](https://github.com/microsoft/vscode-docs/issues/4809) @gregvanl @Tyriar
> 
> ### Diferido
> * [ ]  Permitir el cambio de tamaño del widget de búsqueda en blocs de notas [vscode#141516](https://github.com/microsoft/vscode/issues/141516) @rebornix
> * [ ]  Admite la copia de salida simple al portapapeles [vscode# 129880](https://github.com/microsoft/vscode/issues/129880) @amunger @rebornix
> * [ ]  Mostrar cambios en la regla de información general del editor de diferencias de bloc de notas [vscode#146224](https://github.com/microsoft/vscode/issues/146224)  @amunger @rebornix
> * [ ]  Proporcionar soporte para ir a la prueba / Ir a la prueba [vscode # 126932](https://github.com/microsoft/vscode/issues/126932) @connor4312
> * [ ]  Alinear el widget de búsqueda del editor con el widget de búsqueda del bloc de [notas vscode#144851](https://github.com/microsoft/vscode/issues/144851) @rebornix
> * [ ]  Haga que la extensión eslint sea más accesible para los nuevos [contribuyentes vscode-eslint#1447](https://github.com/microsoft/vscode-eslint/issues/1447) @dbaeumer
> * [ ]  Investigar la compatibilidad con eslint notebook en la web [vscode-eslint#1366](https://github.com/microsoft/vscode-eslint/issues/1366) @dbaeumer
> * [ ]  Borrador de Git Beginner's Docs [vscode-docs#5033](https://github.com/microsoft/vscode-docs/issues/5033) @digitarald @burkeholland

**Comment by ChaseKnowlden** [comment]
Board is now public

**Comment by mehdialiakbar31** [comment]
> This plan captures our work in **June**. This is a 4-week iteration. We will ship in early **July**.
> 
> ## Endgame
> * **June 27, 2022**: Endgame begins
> * **July 1, 2022**: Endgame done
> 
> The endgame details for this iteration are tracked [here](https://github.com/microsoft/vscode/issues?q=is%3Aissue+label%3Aendgame-plan+milestone%3A%22June+2022%22).
> 
> ## Plan Items
> Below is a summary of the top level plan items.
> 
> Legend of annotations:
> 
> Mark	Description
> 	work in progress
> 	blocked task
> 	stretch goal for this iteration
> 	missing issue reference
> 	more investigation required to remove uncertainty
> 	under discussion within the team
> 	a large work item, larger than one iteration
> ### Accessibility
> * [ ]  Accessibility issues, see [query](https://github.com/Microsoft/vscode/issues?q=is%3Aopen+is%3Aissue+label%3Aaccessibility+milestone%3A%22June+2022%22) @isidorn **team**
> 
> ### UX
> * [ ]  runner Improve the discoverability of filtering in tree views [vscode#146806](https://github.com/microsoft/vscode/issues/146806) @misolori @joaomoreno
> * [ ]  muscle runner Explore "Do Not Disturb" mode and its implications on existing scenarios [vscode#144324](https://github.com/microsoft/vscode/issues/144324) @daviddossett @bpasero
> * [ ]  muscle Improve editor group focus states in high contrast themes [vscode#145563](https://github.com/microsoft/vscode/issues/145563) @daviddossett
> 
> ### Workbench
> * [ ]  runner Support profile management [vscode#116740](https://github.com/microsoft/vscode/issues/116740) @sandy081 @sana-ajani @daviddossett
> * [x]  runner Profiles: Support global state isolation per profile [vscode#151490](https://github.com/microsoft/vscode/issues/151490) @bpasero @sandy081
> * [ ]  red_circle muscle Support hiding toolbar items using right click @jrieken
> * [ ]  runner Explore `Continue On...` support in core [vscode#141293](https://github.com/microsoft/vscode/issues/141293#issuecomment-1117468466) @joyceerhl
> * [ ]  runner Explore a `Share as...` menu for creating `vscode.dev` links [vscode#146309](https://github.com/microsoft/vscode/issues/146309) @alexr00
> * [x]  runner muscle Adopt inline suggestions for previewing emmet expansions [vscode#139247](https://github.com/microsoft/vscode/issues/139247) @rzhao271 @hediet
> * [ ]  runner Polish Command Palette launcher in titlebar [vscode#149445](https://github.com/microsoft/vscode/issues/149445) @jrieken @sbatten
> * [ ]  runner Settings Editor: Improve setting override indicators UX when many items are present [vscode#151787](https://github.com/microsoft/vscode/issues/151787) @rzhao271
> * [ ]  runner Settings Editor: Indicate if setting has a language-specific value [vscode#145731](https://github.com/microsoft/vscode/issues/145731)  @rzhao271
> * [ ]  hand Adopt Window Control Overlay API in electron 18 [vscode#147947](https://github.com/microsoft/vscode/pull/147947) @sbatten
> * [x]  runner Enable group policy support for `UpdateMode` [vscode#147756](https://github.com/microsoft/vscode/issues/147756) @joaomoreno
> * [x]  Use merge editor for resolving conflicts in settings sync [vscode#151940](https://github.com/microsoft/vscode/issues/151940) @sandy081
> 
> ### Web
> * [ ]  runner Continue translations in the web [vscode#82595](https://github.com/microsoft/vscode/issues/82595) @TylerLeonhardt
>   
>   * [ ]  Support built-in extension translations in web
>   * [ ]  Support marketplace extenson translations in web
> * [x]  runner Explore how a web extension can leverage the [WebUSB API](https://wicg.github.io/webusb) [vscode#116761](https://github.com/microsoft/vscode/issues/116761) @mjbvz @alexdima
> * [ ]  red_circle Support default themes in `/theme` @aeschli
> * [ ]  runner Explore a partial WASI implementation using the VS Code API [vscode#149092](https://github.com/microsoft/vscode/issues/149092) @dbaeumer
> * [ ]  runner Explore setting COOP and COEP headers on vscode.dev [vscode#137884](https://github.com/microsoft/vscode/issues/137884) @jrieken
> 
> ### Code Editor
> * [ ]  runner Merge editor [vscode#146091](https://github.com/microsoft/vscode/issues/146091) @hediet @jrieken @bpasero
> * [ ]  runner Context menu on the minimap to hide/show it [vscode#148131](https://github.com/microsoft/vscode/issues/148131) @alexdima
> * [ ]  Investigate providing a default snippet for a language [vscode#145929](https://github.com/microsoft/vscode/issues/145929) @jrieken @digitarald
> * [ ]  red_circle Eliminate notification that shows when there are too many folding regions @aeschli
> * [x]  Add command to preview refactorings [vscode#151140](https://github.com/microsoft/vscode/issues/151140) @justschen
> * [ ]  runner  Start exploring using a custom widget for code actions [vscode#152913](https://github.com/microsoft/vscode/issues/152913) @justschen
> 
> ### Comments
> * [ ]  Allow to disable comments altogether [vscode#149176](https://github.com/microsoft/vscode/issues/149176) @alexr00
> * [ ]  muscle Comments view should have ability to filter by resolved/unresolved [vscode#150958](https://github.com/microsoft/vscode/issues/150958) @alexr00
> 
> ### Notebook Editor
> * [x]  runner Retain controller on `saveAs` or `save` of an untitled notebook [vscode#151158](https://github.com/microsoft/vscode/issues/151158) @roblourens
> * [x]  Improve notebook “scrolling up” rendering performance [vscode#148164](https://github.com/microsoft/vscode/issues/148164) @roblourens
> * [ ]  muscle Explore improving files search results for notebooks [vscode#95275](https://github.com/microsoft/vscode/issues/95275) @roblourens
> * [ ]  Investigate support for hot exit in interactive window [vscode#151444](https://github.com/microsoft/vscode/issues/151444) @lramos15 @rebornix
> * [x]  runner Improve selecting a renderer when multiple are registered for the same output mime-type [vscode#148724](https://github.com/microsoft/vscode/issues/148724) @rebornix
> 
> #### Jupyter Notebooks
> * [ ]  runner Explore improve first-run experience for running a Jupyter notebook; [board](https://github.com/orgs/microsoft/projects/359/views/1); @brettcannon @lramos15 @IanMatthewHuff @karrtikr @minsa110
> * [ ]  runner Contributions to the vscode-jupyter extension [vscode-jupyter#10430](https://github.com/microsoft/vscode-jupyter/issues/10430) @greazer **team**
> 
> ### Languages
> #### AnyCode
> * None.
> 
> #### LSP
> * None.
> 
> #### Python
> * [ ]  runner Contributions to the python extension [vscode-python#19286](https://github.com/microsoft/vscode-python/issues/19286) @brettcannon **team**
> * [ ]  Resolve language server restart issues with `pylance` [LSP #1005](https://github.com/microsoft/vscode-languageserver-node/issues/1005) @dbaeumer
> 
> #### JavaScript/TypeScript
> * [ ]  hand Support TS server plugins on web [vscode#140455](https://github.com/microsoft/vscode/issues/140455) @mjbvz
> * [ ]  hand Support TS project-wide intellisense on web [typescript#47600](https://github.com/microsoft/TypeScript/issues/47600) @mjbvz
> * [ ]  hand Explore automatic type acqusition on web [typescript#45314](https://github.com/microsoft/TypeScript/issues/45314) @mjbvz
> 
> #### Markdown
> * [ ]  runner Finalize DropIntoEditor API for creating file and image links [vscode#142990](https://github.com/microsoft/vscode/issues/142990) @mjbvz
> * [ ]  runner Finalize DocumentPasteEditProvider API to support pasting images [vscode#30066](https://github.com/microsoft/vscode/issues/30066) @mjbvz
> 
> #### CSS
> * None.
> 
> #### JSON
> * [ ]  muscle Explore test suite from JSON schema spec [vscode-json-languageservice#120](https://github.com/microsoft/vscode-json-languageservice/issues/102) @aeschli
> 
> ### Debug
> * [x]  Show breakpoint diagnostic information in the breakpoints view [vscode#142870](https://github.com/microsoft/vscode/issues/142870) @roblourens
> * [ ]  Support to pass unescaped arguments in launch.json [vscode#149910](https://github.com/microsoft/vscode/issues/149910) @roblourens @weinand
> * [ ]  runner Improving the UI for "step in targets" [vscode#149871](https://github.com/microsoft/vscode/issues/149871) @connor4312
> * [x]  runner Add keyboard shortcuts to switch debug consoles [vscode#151772](https://github.com/microsoft/vscode/issues/151772) @andreamah
> * [ ]  runner Add debug consoles to `view` quick access menu [vscode#152400](https://github.com/microsoft/vscode/issues/152400) @andreamah
> * [ ]  Add quick access for debug consoles [vscode#151771](https://github.com/microsoft/vscode/issues/151771) @andreamah
> * [ ]  runner Remove special treatment of extension debugging in VS Code (Prototype extension debugging in a clean environment) [vscode#126737](https://github.com/microsoft/vscode/issues/126737) @weinand
> 
> #### DAP
> * [ ]  runner Support to pass unescaped program arguments [debug-adapter-protocol#146](https://github.com/microsoft/debug-adapter-protocol/issues/146) @weinand @roblourens
> * [ ]  runner Add additional location information to StepInTarget [debug-adapter-protocol#274](https://github.com/microsoft/debug-adapter-protocol/issues/274) @connor4312 @weinand
> 
> #### js-debug
> * [ ]  runner muscle Investigate: Smart Step Into [vscode#123879](https://github.com/microsoft/vscode/issues/123879) @connor4312
> * [x]  runner Allow easy toggling between sourcemaps and compiled code [vscode#151412](https://github.com/microsoft/vscode/issues/151412) @connor4312
> * [ ]  muscle Improve breakpoint predictor caching for large projects [vscode-js-debug#1149](https://github.com/microsoft/vscode-js-debug/issues/1149) @connor4312
> * [ ]  muscle Automatically create debug config with a "dry run" [vscode-js-debug#1214](https://github.com/microsoft/vscode-js-debug/issues/1214) @connor4312
> 
> ### Testing
> * None.
> 
> ### Terminal
> * [ ]  runner Prepare to make shell integration the default [vscode#151935](https://github.com/microsoft/vscode/issues/151935) @Tyriar @meganrogge
> * [x]  runner Address issues with terminal reconnection [vscode#151936](https://github.com/microsoft/vscode/issues/151936) @Tyriar
> * [ ]  runner Improve accessibility in the terminal [vscode#151938](https://github.com/microsoft/vscode/issues/151938) @Tyriar
> * [x]  runner Explore contextual actions in the terminal [vscode#151937](https://github.com/microsoft/vscode/issues/151937) @Tyriar
> 
> ### Tasks
> * [ ]  runner Change feature area ownership of tasks [vscode#149060](https://github.com/microsoft/vscode/issues/149060) @meganrogge @Tyriar @alexr00
> * [x]  runner Explore reconnecting tasks on window reload [vscode#117408](https://github.com/microsoft/vscode/issues/117408) @meganrogge
> * [x]  runner Support configurable icons and color for tasks [vscode#145965](https://github.com/microsoft/vscode/issues/145965) @meganrogge
> 
> ### Source Control
> * [x]  runner Full editor can be used as git commit message editor [vscode#30562](https://github.com/microsoft/vscode/pull/30562) @lszomoru
> * [ ]  muscle Investigate: Support input box spell checking [vscode#35571](https://github.com/microsoft/vscode/issues/35571) @lszomoru
> 
> ### API
> * API finalization: [query](https://github.com/Microsoft/vscode/issues?q=is%3Aissue+is%3Aopen+milestone%3A%22June+2022%22+label%3A%22api-finalization%22) @jrieken @mjbvz
> * API proposals: [query](https://github.com/Microsoft/vscode/issues?q=is%3Aissue+is%3Aopen+milestone%3A%22June+2022%22+label%3A%22api-proposal%22) @jrieken @mjbvz
> * [ ]  runner Support snippets in workspace edits [vscode/145374](https://github.com/microsoft/vscode/issues/145374) @jrieken
> * [ ]  runner Continue finalization of remaining proposed notebook API [vscode#149861)](https://github.com/microsoft/vscode/issues/149861) @mjbvz @rebornix @roblourens
> * [ ]  Finalize API to disable the Source Control input field [vscode#150268](https://github.com/microsoft/vscode/issues/150268) @lszomoru
> 
> ### Extensions
> * [x]  Provide a filter to list all deprecated extensions [vscode#151729](https://github.com/microsoft/vscode/issues/151729) @sandy081
> 
> ### Extension Contributions
> * [ ]  GitHub Repositories: [query](https://github.com/microsoft/vscode-remote-repositories-github/milestone/15) @joyceerhl
> * [ ]  Improvements to the Remote extensions [vscode-remote-release#6818](https://github.com/microsoft/vscode-remote-release/issues/6818) @tanhakabir @chrmarti @aeschli
> * [ ]  runner GHPR: Improvements papercuts, selfhosting issues, [query](https://github.com/microsoft/vscode-pull-request-github/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22June+2022%22) @alexr00
> * [ ]  Close the loop between committing and creating a PR [vscode#151589](https://github.com/microsoft/vscode/issues/151589) @alexr00 @lszomoru
> 
> ### Install / Update
> * [ ]  Windows: Unable to update [vscode#147408](https://github.com/microsoft/vscode/issues/147408) @joaomoreno
> * [ ]  Windows: Prevent update service from activating if running as Admin in User install [vscode#148428](https://github.com/microsoft/vscode/issues/148428) @joaomoreno
> * [ ]  runner Integrate with the Windows 11 explorer context menu [vscode#127365](https://github.com/microsoft/vscode/issues/127365) @deepak1556
> 
> ### Engineering
> * [ ]  runner Sandbox: enable extension host communication [vscode#135268](https://github.com/microsoft/vscode/issues/135268) @deepak1556 @alexdima
> * [ ]  runner Review errors to identify which ones are actionable as error telemetry [vscode#145043](https://github.com/microsoft/vscode/issues/145043) @lramos15 **team**
> * [ ]  runner Explore spliting compilation and transpilation into two distinct gulp tasks @jrieken
> * [ ]  runner Explore using `swc` for transpilation [vscode#150025](https://github.com/microsoft/vscode/issues/150025) @jrieken
> * [x]  runner Execute tests in parallel in CI build [vscode#151740](https://github.com/microsoft/vscode/issues/151740) @lszomoru
> * [ ]  muscle Enable `contextIsolation` for all windows [vscode#128099](https://github.com/microsoft/vscode/issues/128099) @bpasero @rzhao271 @deepak1556
> 
> #### Electron
> * [x]  runner Electron 18 update [vscode#145527](https://github.com/microsoft/vscode/pull/145527) @deepak1556
> * [ ]  runner Improve dependency generation for Debian [vscode#13089](https://github.com/microsoft/vscode/issues/13089) @rzhao271
> * [ ]  WCO on Windows don't respond to touch events [electron#34364](https://github.com/electron/electron/issues/34364) @deepak1556
> * [ ]  WCO on Windows occludes dev tools [electron#34129](https://github.com/electron/electron/issues/34129) @deepak1556
> 
> ### Documentation
> * [ ]  runner Improve TS docs [vscode-docs#5205](https://github.com/microsoft/vscode-docs/issues/5205) @gregvanl @isidorn
> * [ ]  Review and update Integrated Terminal [vscode-docs#4809](https://github.com/microsoft/vscode-docs/issues/4809) @gregvanl @Tyriar
> 
> ### Deferred
> * [ ]  Allow resizing for the find widget in notebooks [vscode#141516](https://github.com/microsoft/vscode/issues/141516) @rebornix
> * [ ]  Support copying simple output to the clipboard [vscode#129880](https://github.com/microsoft/vscode/issues/129880) @amunger @rebornix
> * [ ]  Show changes in the overview ruler of the notebook diff editor [vscode#146224](https://github.com/microsoft/vscode/issues/146224) @amunger @rebornix
> * [ ]  Provide support for Go to Test/Go to Test Subject [vscode#126932](https://github.com/microsoft/vscode/issues/126932) @connor4312
> * [ ]  Align editor find widget with the notebook find widget [vscode#144851](https://github.com/microsoft/vscode/issues/144851) @rebornix
> * [ ]  Make the eslint extension more approachable for new contributors [vscode-eslint#1447](https://github.com/microsoft/vscode-eslint/issues/1447) @dbaeumer
> * [ ]  Investigate eslint notebook support in the web [vscode-eslint#1366](https://github.com/microsoft/vscode-eslint/issues/1366) @dbaeumer
> * [ ]  Draft Git Beginner's Docs [vscode-docs#5033](https://github.com/microsoft/vscode-docs/issues/5033) @digitarald @burkeholland
